extensions_admin/pulp_rpm/extensions/admin/contents.py

```diff
@@ -491,16 +491,16 @@ def write_erratum_detail(self, erratum_list, fields=None):
 
         template_data = {
             'id': erratum_meta['id'],
-            'title': erratum_meta['title'],
-            'summary': erratum_meta['summary'],
+            'title': erratum_meta.get('title', None),
```

Comment:
There is no need to specify `None` as a default value, because [a _default_ defaults to `None`](https://docs.python.org/2/library/stdtypes.html#dict.get) %)